docs/general/oasis-network/README.mdx

```diff
@@ -49,5 +49,5 @@ The versatility of the ParaTime layer allows the Oasis Network to expand and gro
 ### Learn more
 
 <DocCard item={
-    findSidebarItem('/general/oasis-network/why-oasis')
+    findSidebarItem('/general/oasis-network/token-metrics-and-distribution')
```

Comment:
I think we can safely remove the "Learn more" section here.

Reply:
Ah, you are right.